raster/r.geomorphon/main.c

```diff
@@ -35,6 +37,77 @@ typedef struct
     CELL *forms_buffer;
 } MULTI;
 
+static const char *form_short_name(const FORMS f)
+{
+    const char *form_short_names[] = {
+        /* skip 0 */
+        [FL] = "FL",
+        [PK] = "PK",
+        [RI] = "RI",
+        [SH] = "SH",
+        [SP] = "SP",
+        [SL] = "SL",
+        [HL] = "HL",
+        [FS] = "FS",
+        [VL] = "VL",
+        [PT] = "PT",
+        [__] = "ERROR",
+    };
+    return (f >= FL && f <= PT) ? form_short_names[f] : form_short_names[__];
+}
+
+static const char *form_long_name(const FORMS f)
+{
+    /*
+     * The list below is the same as in the documentation and the original paper.
+     * The values in ccolors[] are different for PK and PT.
+     */
+    const const char *form_long_names[] = {
```

Comment:
duplicated `const`

(This is the only complain by clang on r.geomorphon module, with this PR.)

Reply:
removed